admin_guide/manage_scc.adoc

```diff
@@ -260,6 +258,13 @@ For example, to allow the *e2e-user* access to the *privileged* SCC, run:
 $ oadm policy add-scc-to-user privileged e2e-user
 ----
 
+[WARNING]
+====
+Granting access to a user works only for the pods. For other resources
```

Comment:
You can only grant access to a user for the pods. For other resources like deployments and StatefulSets, give access to a service account under which the related controller is operated upon.